src/sass/base/_base.scss

```diff
@@ -9,6 +9,18 @@
 html {
   box-sizing: border-box;
   font-size: 62.5%; // 10px / 16px = 62.5% -> 1rem = 10px
+
+  @include respond(large-screen) {
+    font-size: 75%; // 1rem = 12px
+  }
+
+  @include respond(tab-land) {
+    font-size: 56.25%; // 1rem = 9px
+  }
+
+  @include respond(tab-port) {
```

Comment:
what is tab land vs port?

Reply:
They define the orientation of the tablet screens, landscape and portrait. I have considered the general range of these screens for my responsive design which is 900px-1200px landscape and 600px-900px portrait.